pages/who-we-are/organisation-structure.md

```diff
@@ -2,108 +2,11 @@
 title: DTA organisation structure
 layout: base
 permalink: /who-we-are/organisation-structure/
-searchexcerpt: "Please find an overview of high-level organisation structure of DTA."
 ---
-<article id="content" class="content-listing home">
```

Comment:
The test is failing as you have deleted this line - the skip link expects to find a block with id="content" on each page, this helps with accessibility so people can easily find the content on the page.